src/main/resources/org/sagebionetworks/web/client/view/SynapseStandaloneWikiViewImpl.ui.xml

```diff
@@ -10,16 +10,16 @@
     type='org.sagebionetworks.web.client.SageImageBundle'
   />
 
-  <bh:Div>
```

Comment:
Noticed that there was no space above or below wiki content, and the synapse alert was previously shown below the 400px height blank markdown container